File: 88-MergeSortedArray/88-MergeSortedArray.java
// Last updated: 7/14/2026, 2:05:05 PM
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int r=0;
        for(int j=m;j<n+m;j++){
            nums1[j]=nums2[r];
            r++;
        }
        Arrays.sort(nums1);
        
        
    }
}

File: 88-MergeSortedArray/88-MergeSortedArray.py
# Last updated: 7/14/2026, 2:05:12 PM
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        temp = nums1[:m]

# Step 2: Merge and sort
        temp.extend(nums2)
        temp.sort()

# Step 3: Copy back into nums1 to modify it in-place
        for i in range(m + n):
            nums1[i] = temp[i] 

          
       
        